poros/game_results.rb

```diff
@@ -0,0 +1,64 @@
+
+class GameResults
+  
+  def service 
+    GameService.new
+  end
+
+  def game_params_by_name(game_name)
+    params = service.get_games_by_name(game_name)[0]
```

Comment:
get_games_by_name() will return the api response for `api.../games` `where name = game_name`. There are more values returned than needed